homeworks/andrew.markovskiy_MarkoAndrew93/0-intro/GamesProject/cards.html

```diff
@@ -0,0 +1,265 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" href="../GamesProject/css/styles.css">
+        <link rel="preconnect" href="https://fonts.googleapis.com">
+        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+        <link href="https://fonts.googleapis.com/css2?family=Inter:wght@500;700&family=Roboto:wght@400;700&family=Young+Serif&display=swap" rel="stylesheet">
+        <title>GamesProject</title>
+    </head>
+    <body>
+        <header>
+            <section class="header-conteiner">
+                <div class="header-line">    
+                    <a class="headder-logo_img" href="">
+                        <img class="header-logo" src="../GamesProject/img/logo.png" alt="">
+                    </a>
+                    <nav class="header-menu">
+                        <ul class="header-menu_list">
+                            <li class="header-menu_item">
+                                <a class="header-menu_link" href="">Games</a>
+                            </li>
+                            <li class="header-menu_item">
+                                <a class="header-menu_link" href="../GamesProject/index.html">About</a>
+                            </li>
+                            <li class="header-menu_item">
+                                <a class="header-menu_link" href="">Login</a>
+                            </li>
+                        </ul>
+                    </nav>
+                </div>
+            </section>
+        </header>
+        <main>
+            <h1 class="cards-title">Study project "Games"</h1>
+            <div class="conteiner-cards">
+                <div class="conteiner-cards_item">
+                    <div class="conteiner-cards_info">
+                        <img class="cards-img" src=".//img/Rectangle 7.png" alt="">
+                        <div class="container-cards_title">
+                            <h4>Overwatch 2</h4>
+                            <p class="cards-text">
+                                Big changes come to the Overwatch formula in this sequel....
+                            </p>
+                        </div>
+                    </div>
+                    <ul class="conteiner-cards_list">
```

Comment:
The card content might be:
` <p><span class="bold"> Genre </span> some text </p>`